### DISCUSSION TITLE
How to update atoti from 0.6.4 to 0.6.5?

### DISCUSSION THREAD

**Discussion by yerol89** [atoti/atoti#576] [Q&A]
Hello Everyone,

After updating Atoti to latest version, we run the notebook and get an error. Here is the details of error we get:

**partially initialized module 'atoti' has no attribute 'exceptions' (most likely due to a circular import)**

Anybody faced with this issue after updating to new version?

**Comment by CamilleBlanco** [comment]
Hello, 

Have you updated atoti with pip, conda or poetry ?

It looks like the upgrade partially failed, it might happen if you have not deleted your cache of 0.6.4 version. You could try to create a new virtual environment to see if it works better.
